tools/remote_build/include/rbe_base_config.bazelrc

```diff
@@ -12,28 +13,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# bazelrc file for running gRPC tests with Foundry (remote build execution)
-# manually from developer's workstation
+# bazelrc to configure bazel to use gRPC's main RBE instance for remote cache
+# and to upload build results to build event service (for bazel build UI results).
+# Note that remote build and test execution is left unconfigured.
```

Comment:
Based on the description here, does that mean for other Bazel tests in the repo, we only need to include this bazelrc file, then the ResultStore will function? And, is this cross platform? I do see remote_cache below.

Reply:
Yes (or mostly yes), that was the idea for extracting these settings to a separate file.
I might need to do a bit of finetuning here, but generally it's true.
And yes, it should be platform agnostic settings. 

Basically, if you set `--remote_upload_local_results=false` with this file you'll get results in resultstore and if you
 set `--remote_upload_local_results=true` you'll get both results in resultstore and the remote caching.

As a side note, if you want for the "Bazel" in to appear in testfusion (and not just printed to the log), you need a bit of extra setup (configure the bazel_invocations_ids) and if you want results uploaded to biquery, you also need to do that separately.

Reply:
TIL the `remote_upload_local_results` flag. By name, it seems to suggest whether to upload the final results to remote, but it actually refers to all artifacts of each actions: `Whether to upload locally executed action results to the remote cache.`

> As a side note, if you want for the "Bazel" in to appear in testfusion (and not just printed to the log), you need a bit of extra setup (configure the bazel_invocations_ids) and if you want results uploaded to biquery, you also need to do that separately.

Thanks for the pointer. I do plan to improve at least for Python Bazel tests (and hopefully xDS ones down the line). For the end result, I hope to get how RBE presents the test results. It has summery tabs, each runs' log is saved (example [RBE opt C++](https://source.cloud.google.com/results/invocations/59ea68f7-7e15-41c6-8d59-0419dbd07958/targets/%2F%2Ftest%2Fcpp%2Fend2end%2Fxds:xds_end2end_test@poller%3Depoll1/log)).